lib/core/services/navigation/navigation_service.dart

```diff
@@ -10,5 +10,5 @@ abstract class NavigationService {
 
   Future<dynamic> popAllAndPushNamed(String routeName, {Object arguments});
 
-  bool pop({dynamic returnValue});
+  void pop({dynamic returnValue});
```

Comment:
This should remain bool, because pop still returns a bool.

Reply:
I had the service just like you, but this fix is a must in new versions of Flutter. That what I understand.

![Captura de Pantalla 2020-02-18 a la(s) 08 16 00](https://user-images.githubusercontent.com/1505545/74731563-fc19b700-5226-11ea-99e2-9566a2ad868b.png)

Reply:
Oh I didn't know

Reply:
Me either, when I updated to last version of master channel, the compiler let me know haha